.github/workflows/sql-test-workflow.yml

```diff
@@ -0,0 +1,104 @@
+name: SQL Plugin Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'dev-*'
```

Comment:
i don't think we have `dev-` branch? and why 2 separate workflows for sql?

Reply:
I see, then is this needed (or should be renamed) in the upstream repo? we don't use `dev-*` branch, actually all develop and feature branches should contain a slash so we can remove them after they are merged

Reply:
Fixed in 63c51377. Workflow would run on demand only.